app/src/lib/git/rebase.ts

```diff
@@ -132,14 +134,17 @@ class GitRebaseParser {
     const commitSummary = match[1]
     this.currentCommitCount++
 
-    const value = this.currentCommitCount / this.totalCommitCount
+    const progress = this.currentCommitCount / this.totalCommitCount
```

Comment:
Do you think it's worth checking if `totalCommitCount` is zero to avoid passing `Infinity` to `formatRebaseValue`?

Reply:
I'm fine with this based on the tests in be16bd358